global/php/functions.php

```diff
@@ -111,6 +113,7 @@
                                     echo "0";
                                 } ?>;
         const roles = <?php echo json_encode($roles); ?>;
+        const userGroups = <?php echo json_encode($groups); ?>;
```

Comment:
userGroups may indicate the groups of  the user not the groups that are available to users :thinking: i think a better name would be just `groups`

Reply:
i couldn't call it `groups` as that had conflicts on other apps annoyingly, snapshots completely broke with it